kolibri/plugins/audio_mp3_render/assets/src/module.js

```diff
@@ -7,7 +7,7 @@ class AudioMP3Module extends ContentRendererModule {
     return AudioComponent;
   }
   get contentType() {
```

Comment:
Also noticing that this is completely useless as a property, because the registration all happens on the server side... but it seems silly not having it encoded in the JS..